static/js/ducks/instruments.js

```diff
@@ -14,8 +14,16 @@ const FETCH_INSTRUMENT_FORMS_OK = "skyportal/FETCH_INSTRUMENT_FORMS_OK";
 export const fetchInstruments = () =>
   API.GET("/api/instrument", FETCH_INSTRUMENTS);
 
-export const fetchInstrumentForms = () =>
-  API.GET(`/api/internal/instrument_forms`, FETCH_INSTRUMENT_FORMS);
+export const fetchInstrumentForms = (params = {}) => {
+  if (!Object.keys(params).includes("apitype")) {
```

Comment:
Is the default `api_classname` added to maintain backward compatibility or something? Why not just send it without any `apitype` and let the handler return all sorts of instruments?

Reply:
Also watch out if you are changing `apitype` to `apiType` you need to change it here and maybe in a few other places...

Reply:
Yeah I thought keeping the original behavior made sense, but could do something else.

Reply:
If there's no real reason not to return either kind of api then I think we should leave it without a default call with `api_classname`.